src/c/project-contributions-per-location-table.js

```diff
@@ -0,0 +1,57 @@
+window.c.ProjectContributionsPerLocationTable = (function(m, models, h) {
+  return {
+    controller: function(args) {
+      var vm = m.postgrest.filtersVM({project_id: 'eq'}),
+          contributionsPerLocation = m.prop([]);
+
+      vm.project_id(args.resourceId);
+
+      models.projectContributionsPerLocation.getRow(vm.parameters()).then(contributionsPerLocation);
+
+      return {
+        contributionsPerLocation: contributionsPerLocation
+      };
+    },
+    view: function(ctrl) {
+      return m('.project-contributions-per-location', [
+        m('.fontweight-semibold.u-marginbottom-10.fontsize-large.u-text-center', 'Localização geográfica dos apoios'),
+        ctrl.contributionsPerLocation().map(function(contributionLocation){
+          return m('.table-outer.u-marginbottom-60', [
+            m('.w-row.table-row.fontweight-semibold.fontsize-smaller.header', [
+              m('.w-col.w-col-4.w-col-small-4.w-col-tiny-4.table-col', [
+                m('div', 'Estado')
+              ]),
+              m('.w-col.w-col-4.w-col-small-4.w-col-tiny-4.table-col[data-ix="sort-arrows"]', [
+                m('a.link-hidden[href="#"]', [
+                  'Apoios  ',m('span.fa.fa-sort', '.')
+                ])
+              ]),
+              m('.w-col.w-col-4.w-col-small-4.w-col-tiny-4.table-col[data-ix="sort-arrows"]', [
+                m('a.link-hidden[href="#"]', [
+                  'R$ apoiados ',
+                  m('span.w-hidden-small.w-hidden-tiny','(% do total) '),
+                  ' ',m('span.fa.fa-sort', '.')
+                ])
+              ])
+            ]),
+            m('.table-inner.fontsize-small', [
+              contributionLocation.source.map(function(source) {
```

Comment:
I see you are using a native .map implementation here while in other places we use underscore's. In the past we used to have some problems of performance (specially mobile) and support with native array.prototype.map, but don't know if that's still the case. But despite of that, I think we should opt for one of those implementations and use it all across the application. What do you guys think?